pkg/syncer/metadatasyncer.go

```diff
@@ -2909,3 +2915,80 @@ func startStoragePolicyQuotaCRInformer(ctx context.Context, cfg *restclient.Conf
 	}()
 	return nil
 }
+
+// storagePolicyUsageCRSync patches StoragePolicyUsage CRs for k8s PVs in Bound state
+func storagePolicyUsageCRSync(ctx context.Context, metadataSyncer *metadataSyncInformer) {
+	log := logger.GetLogger(ctx)
+	log.Debugf("Starting CnsVolumeOperationRequest CR sync.")
+	// Get K8s PVs in State "Bound".
+	k8sPVsInBoundState, err := getBoundPVs(ctx, metadataSyncer)
+	if err != nil {
+		log.Errorf("Failed to get PVs from kubernetes. Err: %+v", err)
+		return
+	}
+	k8sVolumesToNamespaceMap := make(map[string][]*v1.PersistentVolume)
```

Comment:
Shouldn't this be named `NamespaceToK8sVolumesMap`?

Reply:
Updated